lib/rides/widgets/search_ride_filter.dart

```diff
@@ -197,7 +202,7 @@ class SearchSuggestionFilter {
   Widget _buildDeviationFilter(BuildContext context) {
```

Comment:
while we're at it, can we make this a better input type? It should be a field with up and down buttons for whole numbers (similar to the seat input, but of course different design)